### DISCUSSION TITLE
question about discreteMetaAction

### DISCUSSION THREAD

**Discussion by Cuizy-faiz** [eleurent/highway-env#277] [Q&A]
Hi，@eleurent！DiscreteMetaAction has an attribution target_speeds. I can't understand its meaning. For example, in roundabout_env, Its default value is [0,8,16]. What does each dimension of [0,8,16] mean ? 
Furthermore, I want to apply hierarchical RL on roundabout_env, such as choose metaAction->choose target_speed. How to change the value of  target_speed during training?  Do I have to change the environment configuration every time ?

**Comment by eleurent** [comment]
See answer in https://github.com/eleurent/highway-env/issues/278
